lib/internal/structured_clone.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const { MessageChannel, receiveMessageOnPort } = require('internal/worker/io');
+
+
+function structuredClone(value, transfer) {
+  const channel = new MessageChannel();
```

Comment:
Let's leave a TODO comment in here to find a more efficient approach later that does not require creating a `MessageChannel` and `MessagePort` pair.